lib/rbac/filterer.rb

```diff
@@ -563,20 +561,27 @@ def to_class(klass)
       klass.kind_of?(String) || klass.kind_of?(Symbol) ? klass.to_s.constantize : klass
     end
 
-    def apply_scope(klass, scope)
-      klass = klass.all
-      scope_name = Array.wrap(scope).first
+    def send_scope(klass, scope)
+      scope_name = Array(scope).first
```

Comment:
Should I be using `Array.wrap()` or `Array` here? It seems in this case there's no difference.

Reply:
I don't *think* it makes a difference in this case; I'd probably keep it the same though unless you're sure.

Reply:
you're good. was there a reason to change the code at all?

Reply:
@kbrock, nope, I've just wrote whole function from scratch again and realized it's (nearly) the same afterwards. Nevertheless, I revert to wrap, so the changeset is smaller.